modules/global-classes/module.php

```diff
@@ -12,7 +12,9 @@
 }
 
 class Module extends BaseModule {
-	const EXPERIMENT_NAME = 'global_classes';
+	const NAME = 'global_classes';
+
+	private Global_Classes_REST_API $api;
```

Comment:
Why do we put it on a property?

Reply:
We thought that the name might change, but I will delete it for now